src/Jellyfin.Plugin.Dlna.Model/ResponseProfile.cs

```diff
@@ -59,7 +60,7 @@ public ResponseProfile()
     /// <summary>
     /// Gets or sets the conditions.
     /// </summary>
-    public IReadOnlyList<ProfileCondition> Conditions { get; set; }
+    public ProfileCondition[] Conditions { get; set; }
```

Comment:
Please recheck if this is nessesary in the context of XML serialisation otherwise please revert

Reply:
Object is referenced by DlnaDeviceProfile which is used in the API.